src/ts/components/subtitletogglebutton.ts

```diff
@@ -0,0 +1,63 @@
+import { ToggleButton, ToggleButtonConfig } from './togglebutton';
+import { i18n } from '../localization/i18n';
+import { PlayerAPI } from 'bitmovin-player';
+import { UIInstanceManager } from '../uimanager';
+import { StorageUtils } from '../storageutils';
+import { SubtitleSelectBox } from './subtitleselectbox';
+import { SettingsPanel } from './settingspanel';
+
+
```

Comment:
Please remove one of these empty lines. Weird that the linter does not complain there 🤔